### PR TITLE
align with Chinese document

### PR DESCRIPTION
**Pull request by dbgee**

### CODE REVIEW

File: contents/en/basics/inspiration.md [hunk 1]
```diff
@@ -4,3 +4,4 @@ The following list provides some ideas when you have a question "I don't know ho
 
 - [Apache ECharts Official Examples](${mainSitePath}/examples)
 - "How To Guides - Common Charts" in this handbook
+- Works from makeapie.com community
```
Comment (Ovilia) [inline contents/en/basics/inspiration.md:7]: It should contain the link as Chinese version does. Like `Community works from [makeapie.com](https://www.makeapie.com/)`.

**Review by Ovilia — CHANGES_REQUESTED** [review]
Thanks for your contribution. A little improvement is required.

File: contents/en/basics/inspiration.md (hunk 1 above)
Comment (dbgee) [reply contents/en/basics/inspiration.md:7]: Noted，this is my first time to contribution. Do I need to open a new PR or some better way？

Comment (plainheart) [reply contents/en/basics/inspiration.md:7]: @dbgee You may click [here](https://github.com/dbgee/echarts-handbook/edit/patch-2/contents/en/basics/inspiration.md?pr=%2Fapache%2Fecharts-handbook%2Fpull%2F28
) to continue to make changes and commit to the current branch `patch-2`.

**Review by Ovilia — COMMENTED** [review]
Thanks for your contribution! ヽ(ﾟ∀ﾟ)ﾒ(ﾟ∀ﾟ)ﾉ

**Pull request merged**